mern/client/src/screens/Landing/uploadTranscript.js

```diff
@@ -38,6 +41,9 @@ function UploadTranscript() {
           text={"Go Back"}
         />
       </div>
+      <div class="scroller">
```

Comment:
maybe make a new flow page to test the scroller independently? then we can build the new flow page off of that.